ibm/mas_devops/roles/ocs/defaults/main.yml

```diff
@@ -1,3 +1,6 @@
 ---
 # The device path needed for the local volume used by the local storage operator
 lso_device_path: "{{ lookup('env', 'LSO_DEVICE_PATH') | default('/dev/vdb', True) }}"
+
+# ocs action
+ocs_action: "{{ lookup('env', 'OCS_ACTION') | default('install', True) }}"
```

Comment:
Can we add this new paramter to the documentation?

Reply:
Done